root/static/scripts/edit/components/URLInputPopover.js

```diff
@@ -144,7 +144,7 @@ const URLInputPopover = (props: PropsT): React.MixedElement => {
     <ButtonPopover
       buildChildren={buildPopoverChildren}
       buttonContent={null}
-      buttonProps={{className: 'icon edit-item'}}
+      buttonProps={{className: 'icon edit-item', title: l('Edit Link')}}
```

Comment:
Should we also change "Remove Link"? I used "Link" just because of that :)

Reply:
It removes both the URL and any associated relationship, which are the two meanings associated to “link”, so I’m not sure if the current wording is an issue, but “Remove URL” would probably work too. Note that it would be a new message.